kalitectl.py

```diff
@@ -352,6 +352,11 @@ def manage(command, args=[], in_background=False):
             kwargs = {'creationflags': subprocess.CREATE_NEW_PROCESS_GROUP}
         else:
             kwargs = {}
+
+        kwargs.update({
+            "stdout": open("kalite.log", "a"),
```

Comment:
Log should be configurable?

I guess you can get the pid of the process for putting in the .pid file.. but I'm not sure how we should go about having multiple subprocesses opened..